accessible/clipboard.service.js

```diff
@@ -0,0 +1,98 @@
+var blocklyApp = blocklyApp || {};
+
+blocklyApp.ClipboardService = ng.core
+  .Class({
+    constructor: function() {
+      this.clipboardBlockXml;
```

Comment:
Nope! I mean, no in Javascript. I thought it would be helpful to have them there so that progeny knows what class variables to expect in the rest of the class. Thoughts?